language/move-lang/src/to_bytecode/translate.rs

```diff
@@ -546,8 +602,8 @@ fn exp_(context: &mut Context, code: &mut IR::BytecodeBlock, e: H::Exp) {
     match e_ {
         E::Unreachable => panic!("ICE should not compile dead code"),
         E::UnresolvedError => panic!("ICE should not have reached compilation if there are errors"),
-        E::Spec(_) => (),
         E::Unit => (),
+        E::Spec(id) => code.push(sp(loc, B::Nop(Some(context.nop_label(id))))),
```

Comment:
Looks like we are pushing Some(NopLabel) every time.  So None will be used for some other purpose.  I am wondering what is the use case for Nop(None).

Reply:
Its just for API design. The Nop in the IR might be used just as a normal Nop in some situations.